mycroft.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ ! -z $1 ]
```

Comment:
I think it's considered best practice to enclose the `$1` in double quotes.
eg:
```bash
if [ ! -z "$1" ] 
```

I ran a test to make sure it'll still work as expected with quotes, and it does.
```bash
String="not null"

if [ ! -z "$String" ]
then
  echo "\$String is NOT null."
else
  echo "\$String is null."
fi    
# result: $String is NOT null.
```